pagopa/io-app
# chore(IT Wallet): [SIW-942] Add payments section redirect banner within wallet section

> [!WARNING]
> Depends on https://github.com/pagopa/io-app/pull/5588

## Short description
This PR adds the payments section redirect banner in the wallet section screen. The banner visibility is being persisted to the store, which allows the user to hide it and persist the preference between app launches.
**Note:** the banner do not redirect to the payments section yet, as that section is still being implemented.

## List of changes proposed in this pull request
- Added `WalletPaymentsRedirectBanner`
- Updated `WalletHomeScreen` with the redirect banner
- Added reducer, actions and selector to write and read the banner visibility state
- Added migration the redux-persist

## How to test
Enable the **New wallet section** from the **Human Interface** settings in Profile page. Check that the banner is displayed correctly and, when closed, its state is persisted between app launches.

## Preview

https://github.com/pagopa/io-app/assets/6160324/bf4d898f-9e20-4bba-bef8-c05cc5c5d8aa
